### DISCUSSION TITLE
How to pass JVM arguments to IKVM such as 'user.dir'?

### DISCUSSION THREAD

**Discussion by HelloItMeMort** [ikvmnet/ikvm#599] [Q&A]
Experiencing failing tests when upgrading from pre-revived IKVM to more recent versions, as it seems like Path.get("").toAbsolutePath() is resolving to bin/Debug/net8 instead of where the .csproj is located. We have a large amount of tests and would prefer to modify the default directory for FileSystem/FileSystemProvider to minimize the amount of changes necessary by setting the 'user.dir' property before the JVM is initialized, as setting it during runtime does not seem to work

**Comment by wasabii** [comment]
System.setProperty? This is more of a Java question. 

Checking the Java documentation, it says it's supposed to be the OS working directory. No idea why it would ever have been related to the project file at all. It's an OS property. Where the executable was started from.
